packages/common/src/services/explore/Explore.ts

```diff
@@ -90,10 +90,13 @@ export class Explore {
         id: encodeHashId(currentUserId),
         limit: 100
       })
-      const activityData = history.data as APIActivityV2[]
-      const listenedToTracks = activityData
-        .map(makeActivity)
-        .filter(removeNullable) as UserTrackMetadata[]
+      const listenedToTracks = transformAndCleanList(
```

Comment:
`transformAndCleanList` does remove nullable already?

Reply:
item unfortunately can be null